examples/subscription/mqemitter-mongodb-subscription.js

```diff
@@ -1,6 +1,6 @@
 'use strict'
 
-const GQL = require('fastify-gql')
+const mercurius = require('fastify-gql')
```

Comment:
```suggestion
const mercurius = require('mercurius')
```

or `../..` to match the other examples